modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -37,39 +36,67 @@ This example illustrates:
+Once your application is running, you can trigger the notification by going to your application's URL, for example: +http://YOURAPP.cloudhub.io/hello-notification+. The page displays a `Hello World` message, and automatically creates a new notification in your Runtime Manager console. 
+
+Return to the Runtime Manager console and click the new notification indicator bell in the upper-right to display the alert message:
 
 image::image2014-10-24-17-41-16-1.png[image2014-10-24+17-41-16-1]
 
 If the notification is sent after an exception, it attaches the
 `exception.message` and `exception.stacktrace` as custom properties of the notification, which are accessible from the Runtime Manager console.
 
-== Creating a Custom Alert
-
-Alerts allow you to send emails based on your application notifications. To create a custom alert for your application notification, click your application, and then click *Alerts* from the left menu.
-
-In the *New Alert* dialog:
+== Custom Alerts
 
-. Click *New alert*.
-. Enter an alert name: `Hello World`.
-. Optionally select your application name or leave the selection as *any*.
-. Select *custom application notification* as the event type.
-. Enter `Hello` as the word that triggers your notification. Recall the message you configured above is `Hello World`.
-. Enter email addresses that you wish to receive alert email.
-. Click *Create*.
+Alerts allow you to send emails based on your application's notifications. You can create standard alerts in CloudHub with the software provided. To create custom alerts, you need to get the https://www.mulesoft.com/exchange/#!/cloudhub-integration-connector[CloudHub Connector] and install it in Anypoint Studio.
 
+[NOTE]
+Custom alerts and notifications are available only for applications deployed to CloudHub workers. They are not available for applications that you deploy to on-premises servers. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
+
+=== Prerequisites
+
+* Install and configure the CloudHub Connector. The CloudHub Connector is available for both Mule 3 and Mule 4 applications.
+* xref:studio::to-download-and-install-studio.adoc[Install Anypoint Studio].
+
+=== Create a Custom Alert
+
+. Create a global configuration for the CloudHub Connector to define how to sign into Anypoint Platform. It can use basic authentication or inherited token authentication. 
+. In Anypoint Studio, drag the CloudHub Connector from the Mule palette into the application flow for which you want to trigger an alert. 
+. Create the custom notification inside the application, and complete the connector configuration.
+. Go to Runtime Manager to bind the custom notification to an alert. 
+. Click *Alerts* in the menu on the left. 
+. Click the blue plus sign to create a new alert.
+. In the *Create an Alert* window, configure the information for your custom alert. +
+* Name - Enter a meaningful name for the custom alert. 
```

Comment:
Should Name and Severity level be in bold for UI label?

Reply:
also lines 77, 18, 84, 85, 93, 94, 95: I can't tell if they are in the UI or why they might not have UI bold.